q2_deblur/tests/test_denoise.py

```diff
@@ -101,8 +101,17 @@ def test_defaults(self):
         self.assertEqual(list(stats.columns), STATS_HEADER[1:])
         self.assertEqual(len(stats), 0)
 
+    def test_left_trim_len(self):
+        obs_tab, rep_seqs, stats = denoise_16S(self.demux_seqs, 110,
+                                               left_trim_len=10)
+
+        self.assertEqual(len(obs_tab.ids(axis='sample')), 16)
+        self.assertEqual(len(obs_tab.ids(axis='observation')), 20)
+        self.assertEqual(len(list(rep_seqs)), 20)
```

Comment:
what about making sure that all sequences are trimmed to the correct length?

Reply:
Are we in the business of testing deblur now?

Reply:
Note, that was sarcasm!

Reply:
good point... unnecessary.